data/pizza.json

```diff
@@ -1180,7 +1168,7 @@
 			"hidden_ingredients": [
 
 			],
-			"price": 85
+			"price": 89
 		},
 		"vegetale": {
```

Comment:
[Fårost and no tomatsås](https://sannegarden.com/meny/johanneberg/#:~:text=Champinjoner%2C%20L%C3%B6k%2C%20F%C3%A5rost%2C%20Gr%C3%A4dds%C3%A5s%2C%20Strimlad%20Feferoni%2C%20Tomater)